views/partials/main/header.hbs

```diff
@@ -24,8 +24,8 @@
         </li>
       </ul>
       <form class="form-inline my-2 my-lg-0">
-        <input class="form-control mr-sm-2" type="text" placeholder="Search">
-        <button class="btn btn-secondary my-2 my-sm-0" type="submit">Search</button>
+        <iframe src="https://duckduckgo.com/search.html?width=270&prefill=Search DuckDuckGo&focus=yes&kn=1" style="overflow:hidden;margin:0;padding:0;width:328px;height:40px;"
+          frameborder="0"></iframe>
```

Comment:
Please, make it search only in https://lernwerkstatt-site.azurewebsites.net/ and style it (without creating new CSS) to look the same :)

Reply:
Solved, here is a list with the used url parameters:
https://duckduckgo.com/params